domains/_vercel.amardeep.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+      "username": "amardeeplakshkar",
+      "email": "amardeep.devs@gmail.com",
+      "discord": ""
```

no need field if its empty
```suggestion
      "email": "amardeep.devs@gmail.com"
```